src/styles/blocks/button.css

```
@@ -0,0 +1,19 @@
+.button {
+  margin: 0;
+  appearance: none;
+  display: inline-block;
+  padding: 0.5em 1.25em;
+  border: 1px solid;
+  font: inherit;
+  text-decoration: none;
+  color: var(--color-text);
+  background-color: var(--color-background);
+  border-radius: 6px;
+  cursor: pointer;
+}
+
+.button:active {
```

```suggestion
.button:active {
```

Давай не забудем и состояние фокуса. Тут либо оставить браузерное, либо запросить у дизайнеров хотя бы намёк на то, как будет выглядеть фокус на ссылках и кнопках.

---

Браузерное состояние фокуса я не убирал